doc/nrf/known_issues.rst

```diff
@@ -652,6 +652,17 @@ MCUboot
 Recovery with the USB does not work
   The MCUboot recovery feature using the USB interface does not work.
 
+Trusted-Firmware-M
+******************
+
+.. rst-class:: v1-5-0
+
+Missing files or permissions when building on Windows (SES)
+  Because of the Windows path length limitiations, the build can fail with errors related to permissions or missing files if some paths in the build are too long.
+
+  **Workaround:** Shorten the build folder name, e.g. from "build_nrf5340dk_nrf5340_cpuappns" to "build", or shorten the path to the build folder in some other way.
```

Comment:
This is a Windows platform issue as it also happens when building on command line, so not a specific SES issue.
Please note the error message:
```
CMake Warning in C:/projects/github/ncs/modules/tee/tfm/tf-m-tests/test/test_services/tfm_secure_client_service/CMakeLists.txt:
  The object file directory
    C:/projects/github/ncs/zephyr/samples/tfm_integration/psa_level_1/build_nrf9160dk_nrf9160ns/tfm/test/test_services/tfm_secure_client_service/CMakeFiles/tfm_psa_rot_partition_secure_client_service.dir/./
  has 202 characters.  The maximum full path to an object file is 250
  characters (see CMAKE_OBJECT_PATH_MAX).  
```
[CMAKE_OBJECT_PATH_MAX](https://cmake.org/cmake/help/latest/variable/CMAKE_OBJECT_PATH_MAX.html) 
> ... CMake has a built-in limit for each platform

This path works in SES on Linux:
```
/projects/github/ncs_123134423142134234153tr24535432rfr34w5r342r3425r432r45r4323f4r432tr324f342rt342rf342f342tr32f324tf3425rt/zephyr/samples/tfm_integration/psa_level_1/build_nrf9160dk_nrf9160ns_ses/modules/tfm/tfm-prefix/src/tfm-stamp/tfm-configure
```
without warnings or errors.

So maybe highlight that it is only SES-NE on Windows, not SES-NE in general.

Also does this belong in TF-M known issues ?
It will happen in any project that goes beyond `CMAKE_OBJECT_PATH_MAX` and is that really something we should state as a known issue ?

@shantha-14 @carlescufi should we stated limitations of an OS as a known issue ?

Reply:
>So maybe highlight that it is only SES-NE on Windows, not SES-NE in general.

Isn't this sufficiently highlighted? Windows is mentioned twice.

>Also does this belong in TF-M known issues ?

I placed it here since I thought people would most likely encounter it when enabling BUILD_WITH_TFM and having it suddenly fail. Ideally we should reference it in a SES section as well, but I don't know the best way to do that. @FrancescoSer can you help?

Reply:
> > So maybe highlight that it is only SES-NE on Windows, not SES-NE in general.
>
> Isn't this sufficiently highlighted? Windows is mentioned twice.

Not sure.
Maybe because I started from the commit messages, I read this with a different starting point.
`TFM: Bring in fixes for SES`

At least to me `Windows (SES)` give the feeling that it's related to SES, and not a general Windows issue (also happening in command line).

So maybe drop `(SES)` or highlight that it regardless of tool, `SES-NE`, `west build`, `CMake/ninja`.

Reply:
Moved to a new section "Windows" and dropped all references to SES.

Reply:
@umapraseeda Can you comment on the placement of this known issue?

Reply:
Can this be under Build system?

Reply:
Thanks, perfect!